sdk/deepbook/src/client.ts

```diff
@@ -121,16 +151,28 @@ export class DeepBookClient {
 	}
 
 	/**
-	 * @description: Create and Transfer custodian account to user
-	 * @param currentAddress: current user address, eg: "0xbddc9d4961b46a130c2e1f38585bbc6fa8077ce54bcb206b26874ac08d607966"
+	 * @description: Create Account Cap
+	 * @param txb
 	 */
-	createAccount(currentAddress: string = this.currentAddress): TransactionBlock {
-		const txb = new TransactionBlock();
+	createAccountCap(txb: TransactionBlock = new TransactionBlock()) {
```

Comment:
Don't initialize the transaction block here. You don't return it, so you _have_ to pass it in to use this function